test/test.dart

```diff
@@ -467,13 +467,21 @@ Future<void> tryDeleteRealm(String path) async {
     return;
   }
 
+  final dummy = File("");
+  const duration = Duration(milliseconds: 100);
   for (var i = 0; i < 5; i++) {
     try {
       Realm.deleteRealm(path);
-      await File('$path.lock').delete();
+      
+      //delete lock file
+      await File('$path.lock').delete().onError((error, stackTrace) => dummy);
+
+      //Bug in Core https://github.com/realm/realm-core/issues/5997. Remove when fixed
+      //delete compaction space file
+      await File('$path.tmp_compaction_space').delete().onError((error, stackTrace) => dummy);
+
```

Comment:
```suggestion
      // Delete lock file.
      await File('$path.lock').delete().onError((error, stackTrace) => dummy);

      // Bug in Core https://github.com/realm/realm-core/issues/5997. Remove when fixed.
      // Delete compaction space file.
      await File('$path.tmp_compaction_space').delete().onError((error, stackTrace) => dummy);
```
Just formatting comments.